### PR TITLE
Add in announcement for BNO05 support.

### PR DESCRIPTION
**Pull request by yakmoose**
Just added in a simple announcement for the BNO055 support, not sure as to what level of detail is needed or if the tone is quite right. 

Let me know if you would like any tweaks made and I'll sort them out.

### CODE REVIEW

**Comment by rwaldron** [comment]
Also add yourself here: https://github.com/j5js/johnny-five.io/blob/master/src/authors.json

File: src/news/orientation-bno055.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+date: '2016-02-19 20:30:00'
+status: published
+title: 'BNO055 Sensor and Absolute Orientation'
+slug: 'bno055-sensor-absolute-orientation'
+category:
+  - Announcement
+  - Platform
+---
+
+Do you want to know how your robot is oriented or implement an AHRS but you're not too keen on writing your own sensor fusion algorithm? Johnny-Five now has support for the Bosch BNO055. The BNO055 contains an accelerometer, gyroscope and magentometer on the same die as a Cortex M0 core. The BNO055 performs all the tricky parts of filtering and combining sensor data into usable, stable, absolute Euler angles or quarternions.
```
Comment (rwaldron) [inline src/news/orientation-bno055.md]: Excellent!

**Comment by yakmoose** [comment]
I've added in the authors line, but just want to clean up a few bits in the driver as the calibration needs to be a little more visible to user space

**Comment by rwaldron** [comment]
Let me know when you're ready :)

**Pull request merged**